diff --git a/src/slic3r/GUI/Tab.cpp b/src/slic3r/GUI/Tab.cpp
index 5b7ec39..4a5c185 100644
--- a/src/slic3r/GUI/Tab.cpp
+++ b/src/slic3r/GUI/Tab.cpp
@@ -3527,6 +3527,18 @@ void TabPrint::build()
         optgroup->append_single_option_line("solid_infill_filament");
         optgroup->append_single_option_line("wall_filament");
 
+        optgroup = page->new_optgroup(L("Image map per-layer color (experimental)"), L"param_special");
+        optgroup->append_single_option_line("image_map_per_layer_color_rotation");
+        optgroup->append_single_option_line("image_map_outer_wall_mode");
+        optgroup->append_single_option_line("image_map_wall_offset_distance");
+        optgroup->append_single_option_line("image_map_wall_offset_inward_only");
+        optgroup->append_single_option_line("texture_mapping_outer_wall_gradient_global_strength");
+        optgroup->append_single_option_line("texture_mapping_outer_wall_gradient_max_line_width");
+        optgroup->append_single_option_line("texture_mapping_outer_wall_gradient_min_line_width");
+        optgroup->append_single_option_line("image_map_generic_solver_lookup_mode");
+        optgroup->append_single_option_line("image_map_generic_solver_mode");
+        optgroup->append_single_option_line("image_map_generic_solver_mix_model");
+
         optgroup = page->new_optgroup(L("G-code output"), L"param_gcode");
         optgroup->append_single_option_line("reduce_infill_retraction_mode");
         optgroup->append_single_option_line("gcode_add_line_number");
